app/models/ext_management_system.rb

```diff
@@ -128,6 +128,9 @@ def hostname_uniqueness_valid?
   virtual_column :total_vms_suspended,     :type => :integer
   virtual_total  :total_subnets,           :cloud_subnets
 
+  virtual_total  :connection_definitions, :type => :array
```

Comment:
> Does this work?

@imtayadeway well, the rspec works ...
https://github.com/ManageIQ/manageiq/pull/9502/files#diff-c8b3e6a45e46c1b25dbf4741d3040b59R337